app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,221 @@
+---
+id: first_app.intellij-idea
+title: Building a Java application in IntelliJ IDEA
+slug: learn/intellij-idea
+type: tutorial
+category: start
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+more_learning:
+  youtube:
+    - H_XxH66lm3U
+    - 70_B2DyM8mU
+toc:
+- Installing IntelliJ IDEA
+- Creating a new project
+- Writing and editing code
+- Running your application
+- Testing
+- Debugging
+- Refactoring code
+- Documenting code
+- Searching and navigating
+- Version Control
+- Evolving the project
+- More information
+description: "IntelliJ IDEA is the most used Java IDE and helps you develop your Maven and Gradle Java projects faster."
+last_update: 2024-03-12
+---
+
+An IDE (Integrated Development Environment) allows you to quickly build applications, by integrating a source-code editor with the ability to compile and run your code, as well as integration with any other tools you’ll need for software development, including build tools, testing and debugging tools, version control, databases, and so on. And finally, an IDE will let you search and navigate your codebase in ways your file system won’t.
+
+IntelliJ IDEA is the [most widely used IDE for Java developers](https://www.jetbrains.com/lp/devecosystem-2023/java/#java_ide). It has out-of-the-box integrations with all the tools a Java developer will need for software development, with no need to install any additional plugins to get started.
+IntelliJ IDEA is available in two editions:
+- **_IntelliJ IDEA Community Edition_** - free and open-source, licensed under Apache 2.0. It provides all the basic features for Java and Kotlin development.
```

Comment:
As this would likely be read by beginners, I don't think there's much of a point in including information about the license.
Also, this is a Java site so why mention Kotlin?

Reply:
Already removed (not pushed yet)